lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -58,16 +58,22 @@ def header
         end
       end
 
-    private
-
-      attr_reader :content_item, :current_path
-
-      def first_step_nav
+      def get_active_step_nav
         if @active_step_nav_content_id
           active_step_nav = step_navs.select { |step_nav| step_nav.content_id == @active_step_nav_content_id }
           return active_step_nav.first
+        else
+          return false
```

Comment:
You shouldn't have to return `false` here. If there isn't an active step nav this method should will return `nil` be default, and `nil` evaluates to `false`.